### PR TITLE
fix(clerk-js): Avoid using border-(dir) styles

### PR DESCRIPTION
**Pull request by desiprisg**
## Description

Avoid using `border`, `border-top`, `border-right`, `border-left` and `border-bottom` in favour of directly setting width, style and color to allow for easier overriding by customers.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by octoper** [comment]
!preview

File: packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx [hunk 1]
```diff
@@ -157,7 +157,9 @@ export const CreateOrganizationForm = withCardStateProvider((props: CreateOrgani
                     width: t.sizes.$16,
                     height: t.sizes.$16,
                     borderRadius: t.radii.$md,
-                    border: `${t.borders.$dashed} ${t.colors.$neutralAlpha200}`,
+                    borderWidth: t.borderWidths.$normal,
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx:160]: This should be dashed.

**Review by anagstef — CHANGES_REQUESTED** [review]
1 minor comment

File: packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx [hunk 2]
```diff
@@ -157,7 +157,9 @@ export const CreateOrganizationForm = withCardStateProvider((props: CreateOrgani
...
+                    borderStyle: t.borderStyles.$solid,
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx:161]:
```suggestion
                    borderStyle: t.borderStyles.$dashed,
```

**Comment by octoper** [comment]
!preview

**Comment by anagstef** [comment]
!preview

File: packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx [hunk 3]
```diff
@@ -21,7 +21,9 @@ export const SingleSessionActions = (props: SingleSessionActionsProps) => {
       elementDescriptor={descriptors.userButtonPopoverActions}
       elementId={descriptors.userButtonPopoverActions.setId('singleSession')}
       sx={t => ({
-        borderTop: `${t.borders.$normal} ${t.colors.$neutralAlpha100}`,
+        borderTopWidth: t.borderWidths.$normal,
+        borderBottomStyle: t.borderStyles.$solid,
+        borderBottomColor: t.colors.$neutralAlpha100,
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx:26]: borderTopStyle: t.borderStyles.$solid,
borderTopColor: t.colors.$neutralAlpha100,

**Review by anagstef — CHANGES_REQUESTED** [review]

File: packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx (hunk 3 above)
Comment (anagstef) [inline packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx:26]:
```suggestion
        borderTopStyle: t.borderStyles.$solid,
        borderTopColor: t.colors.$neutralAlpha100,
```

**Review by anagstef — APPROVED** [review]

**Pull request merged**